package com.ilango.Security_project.dto;


import lombok.Builder;
import lombok.Data;


@Data
@Builder
public class AuthResponse {


    private String accessToken;


    private String refreshToken;


    private Long userId;


    private String email;


    private String role;

}